yb_build_thirdparty_main.py

```diff
@@ -958,6 +958,87 @@ def get_openssl_related_cmake_args(self):
         ]
         return openssl_options
 
+class LibTest:
+    """
+    Verify correct library paths are used in installed dynamically-linked eecutables and
+    libraries.
+    """
+    def __init__(self):
+        self.tp_dir = os.path.dirname(os.path.realpath(sys.argv[0]))
+        self.tp_installed_dir = os.path.join(self.tp_dir, 'installed')
+        if is_mac():
+            self.tool = "otool -L"
+            self.okay_paths = "^\t("
+            self.okay_paths += "/usr/|"
+            self.okay_paths += "/System/Library/|"
+            self.okay_paths += "@rpath|"
+            self.okay_paths += "@loader_path|"
+            self.okay_paths += f"{self.tp_dir})"
```

Comment:
It would probably be better to assign these "okay paths" to a list and then join it with `|` once outside of the mac vs. linux `if` statement.